include/crocoddyl/core/activations/quadratic-log.hpp

```diff
@@ -0,0 +1,75 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2018-2019, LAAS-CNRS
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef CROCODDYL_CORE_ACTIVATIONS_QUADRATIC_LOG_HPP_
+#define CROCODDYL_CORE_ACTIVATIONS_QUADRATIC_LOG_HPP_
+
+#include <stdexcept>
+#include "crocoddyl/core/fwd.hpp"
+#include "crocoddyl/core/activation-base.hpp"
+#include "crocoddyl/core/utils/exception.hpp"
+#include <iostream>
+
+namespace crocoddyl {
+
+template <typename _Scalar>
```

Comment:
also here, do not forget to create doxygen documentation for the class, functions and members